Source package : diceware
File name       : diceware-0.9.5.tar.gz
Lookaside URL   : https://src.fedoraproject.org/repo/pkgs/diceware/diceware-0.9.5.tar.gz/sha512/cd7c60d85b48f1fb597b147d029b2adcdb4f130b5822c2de2a50be7b042b704f00e35bc25824fea2edf2ee250f7782e9967ed075659be1ec0d59f18eb7c57105/diceware-0.9.5.tar.gz
Source URL      : https://files.pythonhosted.org/packages/source/d/diceware/diceware-0.9.5.tar.gz

source file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/source-diceware-0.9.5.tar.gz: gzip compressed data, was "dist/diceware-0.9.5.tar", last modified: Sat Apr  7 21:07:05 2018, max compression, original size modulo 2^32 645120
lookaside file type: /tmp/fedora-lookaside-verification--620412-UnEb0Nzwp0B1/lookaside-diceware-0.9.5.tar.gz: gzip compressed data, from Unix, original size modulo 2^32 583680


Only in source-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5: diceware.egg-info
Only in lookaside-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5: .gitignore
Only in source-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5: PKG-INFO
diff -r source-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5/setup.cfg lookaside-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5/setup.cfg
9,13d8
< 
< [egg_info]
< tag_build = 
< tag_date = 0
< 
Only in lookaside-diceware-0.9.5.tar.gz-extracted/diceware-0.9.5: .travis.yml
